<launch>
  <param name="/use_sim_time" value="true" />
  <param name="/starting_x" value="0.0" />
  <param name="/starting_y" value="0.0" />
  <param name="/starting_th" value="0.0" />
  <node pkg="first_project" name="odom_node" type="odom_node" output="screen" />
  <node pkg="tf" type="static_transform_publisher" name="sick_front_left" args="1.85 0.93 0.0 0.81 0.0 3.14 base_link sick_front_left 100" />
  <node pkg="tf" type="static_transform_publisher" name="sick_front_right" args="1.85 -0.93 0.0 -0.76 0.0 3.14 base_link sick_front_right 100" />
  <node pkg="tf" type="static_transform_publisher" name="sick_rear_left" args="-1.85 0.93 0.0 2.38 0.0 3.14 base_link sick_rear_left 100" />
  <node pkg="tf" type="static_transform_publisher" name="sick_rear_right" args="-1.75 -0.8 0.0 -2.30 0.0 3.14 base_link sick_rear_right 100" />
<!-- 
  <group ns="turtlesim1">
    <node pkg="turtlesim" name="sim" type="turtlesim_node"/>
    <node pkg="turtlesim" name="teleop" type="turtle_teleop_key"/>
  </group>

  <group ns="turtlesim2">
    <node pkg="turtlesim" name="sim" type="turtlesim_node"/>
  </group>

  <node pkg="turtlesim" name="mimic" type="mimic">
    <remap from="input" to="turtlesim1/turtle1"/>
    <remap from="output" to="turtlesim2/turtle1"/>
  </node> -->

</launch>
